draft-ietf-rtcweb-jsep.xml

```diff
@@ -1417,10 +1417,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>A rollback is performed by supplying a session
             description of type "rollback" with empty contents to
-            either setLocalDescription or setRemoteDescription,
-            depending on which was most recently used (i.e. if the new
-            offer was supplied to setLocalDescription, the rollback
-            should be done using setLocalDescription as well).</t>
+            either setLocalDescription or setRemoteDescription.
+            The effects MUST be the same regardless of whether
+            setLocalDescription or setRemoteDescription is called. Note
+            that the only reason rollbacks use setLocalDescription
+            and setRemoteDescription rather than an independent method
+            is for compatibility with existing implementations.</t>
```

Comment:
If I were reading this document for the first time, I would be confused by why rollbacks use `setLocalDescription`/`setRemoteDescription`, and would look for an explanation, so personally I think this note assists the reader and doesn't hurt anything. Do you still want it removed? Or would you suggest some alternative wording?